zubhub_frontend/zubhub/src/assets/js/styles/views/faqs/FAQsStyles.js

```diff
@@ -6,9 +6,9 @@ const styles = theme => ({
     paddingTop: '2em',
     paddingBottom: '2em',
     flex: '1 0 auto',
-    background: 'rgba(255,204,0,1)',
+    background: 'var(--primary-color2)',
     background:
-      'linear-gradient(to bottom, rgba(255,204,0,1) 0%, rgba(255,229,133,1) 25%, rgba(255,255,255,1) 61%, rgba(255,255,255,1) 100%)',
+      'linear-gradient(to bottom, var(--primary-color2) 0%, var(--primary-color2) 25%, rgba(255,255,255,1) 61%, rgba(255,255,255,1) 100%)',
```

Comment:
some colors are missing css variables

Reply:
That is white color which we haven't taken any input from in the backend